backend/pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "collective.volto.formsupport==3.2.3",
     "collective.honeypot",
     "plonegovbr.socialmedia==2.0.0a8",
-    "collective.person==1.0.0b2",
+    "collective.person==1.0.0b3",
```

Comment:
@iFlameing Make sure you run `make install` after changing dependencies. There should be a corresponding change in uv.lock (similar to pnpm-lock.yaml but for Python packages)